bigbluebutton-html5/imports/api/audio/client/bridge/livekit.ts

```
@@ -322,6 +355,97 @@ export default class LiveKitAudioBridge extends BaseAudioBridge {
+
+      return newStream;
+    } catch (error) {
+      // Device change failed. Clean up the tentative new stream to avoid lingering
+      // stuff, then try to rollback to the previous input stream.
+      if (newStream && typeof newStream.getAudioTracks === 'function') {
+        newStream.getAudioTracks().forEach((t) => t.stop());
+        newStream = null;
+      }
+
+      // Rollback to backup stream
+      if (backupStream && backupStream.active) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)